fdbclient/SpecialKeySpace.actor.h

```diff
@@ -95,6 +125,8 @@ class SpecialKeySpace {
 	enum class MODULE {
 		CLUSTERFILEPATH,
 		CONNECTIONSTRING,
+		FAILURE, // A single key space contains a json string which describes the last failure in special-key-space
```

Comment:
I think calling this something like error msg instead of failure would make more sense. Failure to me means more like a hardware problem or something than a user error

Reply:
Done. Thanks.